app/models/invoice.rb

```diff
@@ -6,4 +6,8 @@ class Invoice < ApplicationRecord
   has_many :merchants, through: :items
 
   enum :status, ["cancelled", "in progress", "completed"]
-end
\ No newline at end of file
+
+  def self.unshipped_invoices
+    Invoice.joins(:invoice_items).where("invoice_items.status != 2").group("invoices.id").order(:created_at)
+  end
```

Comment:
This finds invoice items that are status = 1 or status = 0, and the `group("invoices.id")` basically just takes out the duplicate invoices